runtime/src/iree/base/internal/threading.h

```diff
@@ -172,6 +172,8 @@ void iree_thread_request_affinity(iree_thread_t* thread,
 // This has no effect if the thread is not suspended.
 void iree_thread_resume(iree_thread_t* thread);
 
+void iree_thread_yield();
```

Comment:
in C you can get better errors for parameter-less functions by making them take void:
```suggestion
void iree_thread_yield(void);
```